### DISCUSSION TITLE
Transferring a layout from one computer to another

### DISCUSSION THREAD

**Discussion by 9a3eedi** [DeaDBeeF-Player/deadbeef#2934] [Q&A]
Hello,

I have spent several minutes fine tuning my own deadbeef layout on my laptop and I got it just the way I like it. Is there a way I can transfer this layout to another computer? Are there any caveats?

Both computers are running Manjaro on Plasma. 

I looked into .config/deadbeef/config .... I found a few lines like gtkui.columns.playlist and gtkui.layout.1.9.0, but I'm not sure if simply copying those lines is the right way of doing it. Is there a better way?

**Comment by Oleksiy-Yakovenko** [comment]
Yeah, I think that's the only way available right now -- to copy the *.layout lines in the config

**Comment by 9a3eedi** [comment]
Is it possible to add a comment to a config file? (that way I don't have to delete the old config in case something goes wrong)

**Reply by begin-theadventure** [reply]
Yes, rename the config, for example: backup-config

**Comment by 9a3eedi** [comment]
I've done this and it worked. Thank you for the help!
